src/tools/run-make-support/src/command.rs

```diff
@@ -167,25 +173,35 @@ impl CompletedProcess {
         self
     }
 
+    /// Checks that trimmed `stdout` does not contain trimmed `content`.
     #[track_caller]
     pub fn assert_stdout_not_contains<S: AsRef<str>>(&self, needle: S) -> &Self {
         assert_not_contains(&self.stdout_utf8(), needle.as_ref());
         self
     }
 
+    /// Checks that trimmed `stdout` contains trimmed `content`.
+    #[track_caller]
+    pub fn assert_stdout_contains<S: AsRef<str>>(&self, needle: S) -> &Self {
+        assert!(self.stdout_utf8().contains(needle.as_ref()));
```

Comment:
Suggestion: I think we really want to expand these asserts to verbosely log the stdout and the expected/unexpected, but that can be done in a follow-up PR.

Reply:
Actually, that's already the case (this PR was not rebased fully, I don't know why it didn't complain about merge conflicts). Only the `assert_x_equals` need verbose logging.

Reply:
I chose to add it right away, as it's fairly trivial.